behaviors/wysiwyg.js

```diff
@@ -440,7 +440,7 @@ function initWysiwygBinder(enableKeyboardExtras) {
         // SelectionAPI R'lyeh fhtagn
         range = document.createRange();
         selection = window.getSelection();
-        range.setStart(editable, 1);
+        range.setStartAfter(editable.lastChild);
```

Comment:
Looks like setStartAfter takes Node not Element https://developer.mozilla.org/en-US/docs/Web/API/Range/setStartAfter